controllers/ocirepository_controller.go

```diff
@@ -942,3 +971,64 @@ func (r *OCIRepositoryReconciler) notify(ctx context.Context,
 		}
 	}
 }
+
+// notify emits notification related to the reconciliation.
```

Comment:
Please explain here what this function does including the keyless fallback

Reply:
OFC, done.